lib/router/index.d.ts

```diff
@@ -278,6 +278,8 @@ declare namespace createRouter {
 
         use(...handlers: HandlerArgument[]): this;
         use(mountPoint: string, ...handlers: HandlerArgument[]): this;
+        use(mountPoint: RegExp, ...handlers: HandlerArgument[]): this;
+        use(mountPoint: (string | RegExp)[], ...handlers: HandlerArgument[]): this;
```

Comment:
Can we just make this one union instead of many overloads? If possible, this would be good to re-use from the router too (perhaps we can make a `type Path`?).